tests/utils.py

```diff
@@ -0,0 +1,37 @@
+from django.db import connection
+
+
+class QueryCount:
```

Comment:
I think I originally introduced this class when I was unaware of a built-in `TransactionTestCase` (Django's base `TestCase` class) way of doing it: https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.TransactionTestCase.assertNumQueries.

We could replace our use of this class like so:

**Old Style**
```python
queries = QueryCount()

with queries.start_count():
    baker.make(models.LonelyPerson, _quantity=5, _bulk_create=True)
    assert queries.count == 6
```

**With built-in `assertNumQueries`**
```python
with.assertNumQueries(6):
    baker.make(models.LonelyPerson, _quantity=5, _bulk_create=True)
```

Reply:
Yes, we can refactor this to use django's `assertNumQueries`. Can you open an issue documenting it and label it as a good first issue?